src/output_panel.c

```diff
@@ -18,6 +18,13 @@
 
 #include "tg.h"
 
+// Zoom slider ranges from 1 to 100
+static const double zoom_min = 1, zoom_max = 100, zoom_mid = (zoom_min + zoom_max)/2;
```

Comment:
My compiler didn't like the assignment of 'zoom_mid = (zoom_min + zoom_max)/2'. It didn't like a variable being used to initialise a 'const'

`
// Zoom slider ranges from 1 to 100
static const double zoom_min = 1, zoom_max = 100;
static const double zoom_mid = (1 + 100)/2;
`

Reply:
What compiler is that?  Gcc even in pedantic ANSI mode accepts it just fine, so it's probably an error in the compiler.

Reply:
@xyzzy42 Looks like gcc6.3.0:

```
... snip
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/local/libexec/gcc/x86_64-apple-darwin15.6.0/6.3.0/lto-wrapper
Target: x86_64-apple-darwin15.6.0
Configured with: ../gcc-6.3.0/configure --enable-languages=c++,fortran
Thread model: posix
gcc version 6.3.0 (GCC) 
configure:2798: $? = 0
```

Here is output of the ./configure

```
 % ./configure   
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ./install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for pthread_mutex_init in -lpthread... yes
checking for sqrt in -lm... yes
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for gtk+-3.0 glib-2.0... yes
checking for portaudio-2.0... yes
checking for fftw3f... yes
checking for windres... no
checking if gcc supports -Wl,--as-needed flag... 
checking if gcc supports -Wall flag... yes
checking if gcc supports -Wextra flag... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating icons/Makefile
config.status: executing depfiles commands

        Tg 0.5.2
        =====

        prefix:                 /usr/local

        compiler:               gcc
        cflags:                 -g -O2 -Wall -Wextra 
        ldflags:        
```

Happy to try stuff if you can direct.

Reply:
Looked into this more.  While perfectly fine in C++, the ability to use const variables in initializer constant expressions in C, is an extension that was added in gcc 8, [Why “initializer element is not a constant” is… not working anymore?](https://stackoverflow.com/questions/54135942/why-initializer-element-is-not-a-constant-is-not-working-anymore).  Since not strictly following the rules on initializer constant expressions is not a required diagnostic, it's not required for the pedantic mode to generate an error.